src/components/Input.tsx

```diff
@@ -0,0 +1,24 @@
+import { useState} from 'react';
+import Button from './Button';
+
+
+const Input=()=> {
```

Comment:
The name Input is too generic in my opinion since we got both `<input/>` itself and `<Button/>` inside.

You should:
1) move `Button` out of this component and place it in `Practice3` instead. Or 
2) rename this component to be something more specific like `SearchInput`

(I understand that this button currently have no real functionality yet but please keep that in mind!)

Reply:
Renamed the component to SearchInput